pages/comparison.vue

```diff
@@ -0,0 +1,70 @@
+<template>
```

Comment:
This page is, obviously, not the final product, it's just a page to test that we are setting the app store values correctly.